engine/test/unit/server/test_compression.py

```diff
@@ -4,7 +4,7 @@
 import gzip
 import bz2
 import lzma
-import lz4.frame
+import lz4.frame  # type: ignore
```

Comment:
And why here? It looks like libs are not updating its type specs with mypy versions, but we should keep the usage of type:ignore as reduced as possible, so please confirm the reason on each different case. You are not supossed to introduce changes to files not affected by the functionllity so I would recommend openining a new PR to deal with mypy problems for existing/untouched code.